__tests__/app.test.js

```diff
@@ -34,7 +34,59 @@ describe('GET /api/topics', () => {
       .get('/api/topics')
       .expect(200)
       .then(({ body }) => {
+        //Check that body contains a 'topics' key
         expect(body).toHaveProperty('topics')
+        expect(Array.isArray(body.topics)).toBe(true)
+        //Check that each topic has 'slug' and 'description' properties
+        body.topics.forEach((topic)=>{
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String)
+            })
+          )
+        })
       })
   })
 })
+
+describe('GET /api/articles/:article_id', () => {
+  test('200: Responds with an article object with the correct properties', () => {
+    return request(app)
+      .get('/api/articles/1') // Replace 1 with a valid article_id from test data
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: 1, // Ensure it matches the requested article_id
```

Comment:
Nice! Well done for making sure the test is still flexible but that we definitely have received back the correct article regardless of what the article info is!